homeassistant/components/group/sensor.py

```diff
@@ -345,13 +354,30 @@ def __init__(
 
     async def async_added_to_hass(self) -> None:
         """When added to hass."""
+        for entity_id in self._entity_ids:
+            if self.hass.states.get(entity_id) is None:
+                self.calculate_attributes_later = async_track_state_change_event(
+                    self.hass, self._entity_ids, self.calculate_state_attributes
+                )
```

Comment:
If the state of two or more entities is None, we will call multiple times `async_track_state_change_event` but only store the last call.

If one entity is not available, we can skip checking the others

```suggestion
                )
                break
```